### ISSUE TITLE
Game cannot be launched within Steam

### ISSUE THREAD

**Issue by nataliavelez** [GooborgStudios/synglechance#31]
Thanks for making this port—I've been looking forward to playing OneShot! I followed the instructions in the latest release to install OneShot for Mac. I install OneShot through SteamCMD, downloaded the archive in the releases page, and ran install.command with seemingly no errors. When I open Steam, the icon for OneShot is highlighted, as it is for any other installed game:

![image](https://user-images.githubusercontent.com/9062288/37788256-2b95ce90-2dbe-11e8-9ca8-e5c8291f4d3f.png)

However, when I try to launch the game, I get this error:
![image](https://user-images.githubusercontent.com/9062288/37788292-4689d53e-2dbe-11e8-94b7-2db70bde4283.png)

Any suggestions on how to fix this?

**Comment by pokedart9001** [comment]
Correct me if I'm wrong, but the reason you're not able to open it via Steam is that the official Steam release is still Windows-only, so while OneShot.app exists, it's not actually integrated with Steam (at least not yet). The reason why a Steam copy is necessary in the first place is just to make sure people aren't pirating the game.
tldr; You're not supposed to be able to open this port via Steam (yet).

**Comment by vinyldarkscratch** [comment]
@pokedart9001 is correct.  This is an issue that we won't be able to fix just yet as this port is unofficial.  Until it's been released on Steam, please use the shortcut placed in your home directory > Applications folder.  (You can use the Launchpad with ⌘⌥L, or Spotlight with ⌘Space, to quickly access it.)  :wink:
